<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="6">
    <widget class="QListWidget" name="SearchSTORED"/>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="SearchEXIT">
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>CONTACT DETAIL</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>SearchNAME</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>NickName</string>
       </property>
       <property name="buddy">
        <cstring>SearchNICKNAME</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>PhoneNumber</string>
       </property>
       <property name="buddy">
        <cstring>SearchPHONENUM</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="SearchNICKNAME"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="SearchPHONENUM"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address</string>
       </property>
       <property name="buddy">
        <cstring>SearchADDRESS</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="SearchADDRESS"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="SearchNAME"/>
     </item>
    </layout>
   </item>
   <item row="2" column="2" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>498</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>412</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>STORED CONTACTS</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>553</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>18</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SearchSTORED</tabstop>
  <tabstop>SearchNAME</tabstop>
  <tabstop>SearchNICKNAME</tabstop>
  <tabstop>SearchPHONENUM</tabstop>
  <tabstop>SearchADDRESS</tabstop>
  <tabstop>SearchEXIT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
